manage_breast_screening/clinics/models.py

```diff
@@ -88,8 +91,12 @@ def completed(self):
             .values("state")[:1]
         )
 
+        completed_last_seven_days = self.filter(
```

Comment:
We've already defined `last_seven_days` on line 79. We just need to add the completed stuff to that I think.